woodwork/serializers/parquet_serializer.py

```diff
@@ -26,6 +39,69 @@ def serialize(self, dataframe, profile_name, **kwargs):
         return super().serialize(dataframe, profile_name, **kwargs)
 
     def write_dataframe(self):
-        file = self._get_filename()
-        dataframe = clean_latlong(self.dataframe)
-        dataframe.to_parquet(file, **self.kwargs)
+        # Create a pyarrow table for pandas, skip for Dask/Spark
+        if isinstance(self.dataframe, pd.DataFrame):
+            dataframe = clean_latlong(self.dataframe)
+            self.table = pa.Table.from_pandas(dataframe)
+
+    def write_typing_info(self):
+        loading_info = {
+            "location": self.location,
+            "type": self.format,
+            "params": self.kwargs,
+        }
+        self.typing_info["loading_info"].update(loading_info)
+        # For Dask and Spark we only get the WW metadata because we haven't created
+        # the pyarrow table yet, but for pandas we combine the existing parquet
+        # metadata with the WW metadata.
+        if _is_dask_dataframe(self.dataframe) or _is_spark_dataframe(self.dataframe):
+            metadata = {
+                "ww_meta".encode(): json.dumps(self.typing_info).encode(),
+            }
+        else:
+            table_metadata = self.table.schema.metadata
+            metadata = {
+                "ww_meta".encode(): json.dumps(self.typing_info).encode(),
+                **table_metadata,
```

Comment:
Question: Here, we're writing the `table_metadata` into the metadata dictionary, but in `deserialize.configure_deserializer`, we only look at the `ww_meta` value. Do we read/look for the `table_metadata` keys/values elsewhere?

Reply:
We don't do anything with the `table_metadata` inside Woodwork, but there is other information stored in there that is needed for deserialization into a pandas DataFrame. If we don't include it here, deserialization breaks.

Reply:
Mm, that's interesting. Thanks for the explanation